docs/components/plugins/api/APISectionClasses.tsx

```diff
@@ -44,7 +44,7 @@ const renderClass = (clx: ClassDefinitionData, exposeInSidebar: boolean): JSX.El
   const Header = exposeInSidebar ? H2Nested : H4;
 
   const isSensorClass = name.endsWith('Sensor');
-  const className = isSensorClass ? name.replace('Sensor', '') : name;
+  const className = isSensorClass && name !== 'LightSensor' ? name.replace('Sensor', '') : name;
```

Comment:
Maybe having a map of names would be better and more extensible in the future? `renderClass` should stay module-name-agnostic (be fully generic) imho.

Reply:
Yeah, we definitely should consider having a names map or rename the page/module to a name which better suits the pattern used for other sensors, like `AmbientLightSensor` and export it as `AmbientLight`, but I did not want to crate braking changes due to docs, since I would like to have the autogenerated sensors package as a part of SDK 47 docs.